cnudie/retrieve.py

```diff
@@ -485,11 +485,30 @@ def lookup(
     return lookup
 
 
+def status_code_if_not_found(image_reference: str) -> int:
+    oci_registry_type = om.OciRegistryType.from_image_ref(
+        image_reference=image_reference,
+    )
+
+    if oci_registry_type in (om.OciRegistryType.GAR, om.OciRegistryType.GCR):
+        return 200
```

Comment:
really sad this mapping is required considering there is an open specification ...
-> https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints